fixture/utf8attrs/.zattrs

```diff
@@ -0,0 +1 @@
+{"foo": "た"}
```

Comment:
This file is missing from the `sdist` on PyPI

Reply:
Thanks, @jakirkham. Argh, I thought I had caught these issues but likely only if there's code to generate it them as well. Are you already working on a fix or shall I?

Reply:
Feel free to go ahead. I might not get to this for a while

Reply:
Forget if we already discussed this before, but maybe we can make some fixes to CI to ensure we catch these issues in PRs. Wrote up some thoughts on this in issue ( https://github.com/zarr-developers/zarr-python/issues/1347 )